e2e/cypress/integration/messaging/message_spec.js

```diff
@@ -137,4 +138,11 @@ describe('Message', () => {
             cy.get(divPostId).should('have.class', 'post--system');
         });
     });
+
+    it('M23361 Focus remains in the RHS text box', () => {
+        cy.postMessage(MESSAGES.MEDIUM);
+        cy.clickPostCommentIcon();
+        cy.postMessageReplyInRHS(MESSAGES.SMALL);
```

Comment:
This is missing "4. Click on Preview" step.

Reply:
Hm...I don't see a "Click on preview" button or icon.  The MM docs does not mention anything on previewing a reply.  Please advise.

https://docs.mattermost.com/help/messaging/sending-messages.html#replying-to-messages